### ISSUE TITLE
No way to explicitly store Filtered messages only or Filtered and Errors. 

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#1299]
Seems like this would be necessary.  For my use cases, I'm storing Errors because I need to investigate and Filtered because I want to do some followup work on them, fix them or finally ignore them.  

Having the ability to do this:

[ ] Store message data
    Filtered:    [ ]     Transformed:  [ ]
    Error:       [ ]     Sent:         [ ]

    ( ) Store indefinitely  ( ) Prune message after [   ] day(s)

Basically spell out what they want to store by type.  Let the users decide.  Maybe, for ease of use, when they check Store Message data for the first time, do the default, which now is checking "With Errors only"

     

Imported Issue. Original Details:
Jira Issue Key: MIRTH-1299
Reporter: garyt
Created: 2010-01-11T09:40:57.000-0800

**Comment by rbeckman-nextgen** [comment]
While processing, every status has to be stored during processing for message durability.  Look into removing specific statuses automatically after message completion.

Imported Comment. Original Details:
Author: jacobb
Created: 2012-11-30T16:23:13.000-0800

**Comment by rbeckman-nextgen** [comment]
3.0 already has an option to remove content upon message completion, so messages that do not have errors or queues will have their content immediately removed after processing.

We do not have a way yet in 3.0 to remove only filtered messages.

Imported Comment. Original Details:
Author: jacobb
Created: 2013-04-08T14:44:49.000-0700
